int16 can_id
int16 HS_temp
int16 motor_temp
int16 voltage_angle
